nix/index.tt

```diff
@@ -26,7 +26,7 @@
     Nix</strong> is to run the following shell command (as a user
     other than <tt>root</tt>):</p>
 
-    <pre><span class="nix-command">curl https://nixos.org/nix/install | sh</span></pre>
+    <pre><span class="nix-command">curl --proto '=https' --tlsv1.2 <nobr>https://nixos.org/nix/install</nobr> | sh</span></pre>
```

Comment:
The `<nobr>` element seems [obsolete](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr).